plugins/inputs/jboss/jboss_test.go

```diff
@@ -0,0 +1,1561 @@
+package jboss
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+var jboss_exec_mode_out = `
```

Comment:
Please move those to some files in a `testdata` directory and read them from there.